omf-python/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license = "MIT"
 [dependencies]
 omf = { path = ".." }
 pyo3 = { version = "0.22.2", features = ["extension-module"] }
+serde_json = "1.0"
```

Comment:
What's this for?

Reply:
We can remove this. I noticed that a lot of the classes have Serializable traits, so I figured a fast way to find out what data was in them was to add a method like:

```rs
    #[getter]
    fn data_json(&self) -> PyResult<String> {
        let data = serde_json::to_string(&self.inner.data)
            .map_err(|e| PyErr::new::<PyValueError, _>(e.to_string()))?;
        Ok(data)
    }
```

which is really handy for development, but probably not needed later, I'll just make a note locally so I can refer back to it and then remove it

Reply:
It's still there. I guess if you want you could move it to dev-dependencies

Reply:
Although, having said that, it is actually useful for the metadata field on Attributes at the moment:

```rs
    #[getter]
    fn metadata(&self) -> PyResult<String> {
        let metadata = serde_json::to_string(&self.inner.metadata)
            .map_err(|e| PyErr::new::<PyValueError, _>(e.to_string()))?;
        Ok(metadata)
    }
```

There should be an easy way to replace this and return a PyDict... I'll have a quick look

Reply:
Happy to merge this as is if you like